src/vs/workbench/services/search/common/search.ts

```diff
@@ -145,6 +145,13 @@ export interface IPatternInfo {
 	isMultiline?: boolean;
 	isUnicode?: boolean;
 	isCaseSensitive?: boolean;
+	notebookInfo?: INotebookPatternInfo;
+}
+
+export interface INotebookPatternInfo {
+	isNotebookMarkdownInput?: boolean;
```

Comment:
I don't understand the names, is it like `isNotebookMarkdownInputEnabled`?

Reply:
I changed the naming, but yes, it's whether we search the notebook's cell source on markdown cells.